# POTD (Pick of the Day) - Freddie Freeman

### Random Forest:

In [1]:
import pandas as pd

rf = pd.DataFrame([['Ronald Acuna', 0.303], 
                   ['Matt Joyce', 0.296], 
                   ['A.J. Pollock', 0.294], 
                   ['Freddie Freeman', 0.289], 
                   ['Nelson Cruz', 0.288]],
columns= ['Player', 'Probability']
)

print(rf)

            Player  Probability
0     Ronald Acuna        0.303
1       Matt Joyce        0.296
2     A.J. Pollock        0.294
3  Freddie Freeman        0.289
4      Nelson Cruz        0.288


### Logistic Regression:

In [2]:
lr = pd.DataFrame([['Freddie Freeman', 0.354], 
                   ['Miguel Cabrera', 0.34], 
                   ['Francisco Lindor', 0.34], 
                   ['Leury Garcia', 0.31], 
                   ['Wilson Ramos', 0.308]],
columns= ['Player', 'Probability']
)

print(lr)

             Player  Probability
0   Freddie Freeman        0.354
1    Miguel Cabrera        0.340
2  Francisco Lindor        0.340
3      Leury Garcia        0.310
4      Wilson Ramos        0.308


#### Freddie Freeman

Freeman will face Stephen Strasburg. He is 18/53 (.340) against him with a handedness advantage.